app/src/main/java/org/pesmypetcare/mypetcare/services/medication/MedicationManagerAdapter.java

```diff
@@ -89,7 +89,7 @@ public void updateMedicationKey(User user, Pet pet, String newDate, String oldDa
             MedicationData medicationData = null;
             try {
                 medicationData = ServiceLocator.getInstance().getPetCollectionsManagerClient()
-                        .getMedication(user.getToken(), user.getUsername(), pet.getName(), oldDate);
+                        .getMedication(user.getToken(), user.getUsername() ,pet.getName(), oldDate);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [',' is preceded with whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5643405)